src/main/java/net/pms/PMS.java

```diff
@@ -897,6 +898,13 @@ public static PMS get() {
 		return instance;
 	}
 
+	@Nonnull
+	public static PMS getNew() {
```

Comment:
Maybe it would be better to name it `getNewInstance`

Reply:
Done